lib/beaker/hypervisor/openstack.rb

```diff
@@ -220,16 +220,15 @@ def provision
 
       @hosts.each do |host|
         ip = get_ip
-        host[:vmhostname] = ip.ip.gsub '.','-'
-        host[:vmfqdn] = host[:vmhostname] + '.rfc1918.puppetlabs.net'
```

Comment:
@tvpartytonight  I'm not as in touch with the community as I'd like to be, but in speaking to the folks I know who use both puppet and openstack it sounds like 'no'. I'm comfortable removing it (especially since it doesn't look very community friendly to begin with).